sentry_sdk/integrations/django/asgi.py

```diff
@@ -0,0 +1,31 @@
+"""
+Instrumentation for Django 3.0
+
+Since this file contains `async def` it is conditionally imported in
+`sentry_sdk.integrations.django` (depending on the existence of
+`django.core.handlers.asgi`.
+"""
+
+from sentry_sdk import Hub
+from sentry_sdk._types import MYPY
+
+from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.asgi import SentryAsgiMiddleware
+
+if MYPY:
+    from typing import Any
+
+
+def patch_django_asgi_handler_impl(cls):
+    # type: (Any) -> None
+    old_app = cls.__call__
+
+    async def sentry_patched_asgi_handler(self, scope, receive, send):
+        # type: (Any, Any, Any, Any) -> Any
```

Comment:
I'm not very familiar with the type annotations. What's the gain of doing it in this file with every type being `Any`?

Reply:
That's just my way of opting out of type annotations. I just feel like it's not worth the effort here bc of the interaction with django, a module unknown to mypy.

Reply:
What would be the difference if this line is not there altogether?

Reply:
I configured mypy to complain about missing type annotations. This way a contributor is forced to think about types of parameters but can still opt out relatively easily. If mypy wasn't configured that way it would assume all parameters and return values to be `Any` on functions that do not have type annotations. This way it's at least explicit.

Reply:
Thanks for explaining :)